content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -139,7 +139,9 @@ Uncheck the **Preview** checkbox:
 
 Finally, click **Save**.
 
-Note that this prevents Preview deployments from working, as they have no access to the database. You can resolve this by creating a separate database for Preview deployments and setting a different `DATABASE_URL` env var, and exposing it only to Preview deployments.
+Note that this prevents Preview deployments from working, as they have no access to the database.
+
+In order to enable Preview deployment for pull requests, create a separate database for preview deployments, set the `DATABASE_URL` env var , and expose it **only** to Preview deployments. **Note that because each pull request shares the same preview database, if you have multiple pull requests with diverging schema migrations, you will encounter problems applying them.**
```

Comment:
```suggestion
In order to enable Preview deployment for pull requests, create a separate database for preview deployments, set the `DATABASE_URL` env var , and expose it **only** to Preview deployments. 

> **Note**: because each pull request shares the same preview database, if you have multiple pull requests with diverging schema migrations, you will encounter problems applying them.
```